src/arfs/bundler.ts

```diff
@@ -0,0 +1,36 @@
+import { DataItem } from 'arbundles';
+import axios from 'axios';
+
+interface BundlerParams {
+	bundlerUrl: URL;
+	isDryRun: boolean;
+}
+
+export class Bundler {
+	private bundlerUrl: URL;
+	private isDryRun: boolean;
+
+	constructor({ bundlerUrl, isDryRun }: BundlerParams) {
+		this.bundlerUrl = bundlerUrl;
+		this.isDryRun = isDryRun;
+	}
+
+	private get dataItemEndpoint(): string {
+		return `${this.bundlerUrl.href}v1/tx`;
+	}
+
+	async sendDataItems(dataItems: DataItem[]): Promise<void> {
+		if (!this.isDryRun) {
+			for (const dataItem of dataItems) {
+				await axios.post(this.dataItemEndpoint, dataItem.getRaw(), {
+					headers: {
+						'Content-Type': 'application/octet-stream'
+					},
+					timeout: 100000,
```

Comment:
Not sure why this value is here. This will timeout in 100 seconds

```suggestion
					timeout: 0,
```

Reply:
[0a22cb1](https://github.com/ardriveapp/ardrive-core-js/pull/216/commits/0a22cb18244703de723bbe1b3ec8521065edef32)